tests/Resources/Page/OCPDashboard/Pods/Pods.robot

```diff
@@ -132,7 +132,30 @@ Verify Container Image
     Should Be Equal  ${image}  ${expected-image-url}
 
 Search Pod
-    [Documentation]   Return list of pods   pod_start_with = here ypu have to provide starting of pod name
+    [Documentation]   Returns list pod  ${pod_start_with} = here ypu have to provide starting of pod name
+    [Arguments]   ${namespace}  ${pod_start_with}
+    ${pod} =  Run  oc get pods -n ${namespace} -o json | jq '.items[] | select(.metadata.name | startswith("${pod_start_with}")) | .metadata.name'
+    @{list_pods} =  Split String  ${pod}  \n
+    [Return]  ${list_pods}
+
+Run Command In Container
+    [Documentation]    Executes a command in a container.
+    ...    If ${container_name} is omitted, the first container in the pod will be chosen
+    [Arguments]    ${namespace}    ${pod_name}    ${command}    ${container_name}=${EMPTY}
+    IF    "${container_name}" == "${EMPTY}"
+        ${output}    Run    oc exec ${pod_name} -n ${namespace} -- ${command}
+    ELSE
+        ${output}    Run    oc exec ${pod_name} -n ${namespace} -c ${container_name} -- ${command}
+    END
+    [Return]    ${output}
+
+Kill Container In Pod
+    [Documentation]  Kills abruptly a running container in a pod executing `bin/kill 1` inside it 
+    [Arguments]  ${namespace}  ${pod_name}  ${container_name}
+    Run    oc exec ${pod_name} -n ${namespace} -c ${container_name} -- /bin/kill 1
+
+Find First Pod By Name
```

Comment:
all the lines of this keyword are implemented by `Search Pod` keyword. I think you can reuse it and return only the list cell you need to return